manager/manager.go

```diff
@@ -23,15 +24,21 @@ type Manager struct {
 	strict     bool
 	db         *gorm.DB
 	uuid       string
+	httpClient http.Client
 }
 
 func NewManager(cacheRoot string, configFile string, strict bool, db *gorm.DB, uuid string) *Manager {
+	client := http.Client{
+		Timeout: time.Second * 10,
```

Comment:
This client is used for doing the remote sha checking. Is 10 seconds a safe amount of time to wait for a response?